### ISSUE TITLE
Compat Macros Shows Misleading Information

### ISSUE THREAD

**Issue by jpmedley** [mdn/yari#6756]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Push_API#browser_compatibility

### What specific section or headline is this issue about?

Browser Compatibility

### What information was incorrect, unhelpful, or incomplete?

The compat section shows multiple tables to cover each of the interfaces that are part of the push api. However, the autogenerated heading includes the interface's parent key within the browser compatibility database.

This will look to someone unfamiliar with the BCD data like a JavaScript object and member reference. In other words, they might assume they can type `api.PushEvent` or something similar in their code.

![image](https://user-images.githubusercontent.com/8204171/180495357-cd62faac-3b0e-4b00-9127-3ba4749177e2.png)

### What did you expect to see?

Instead of something like "api.PushEvent", I should just see "Push Event".

### Do you have anything more you want to share?

Readers should not need to familiarity with MDN's data storage mechanisms to interpret what's on an MDN page.

**Comment by caugner** [comment]
This issue arised by allowing multiple BCD queries per page (https://github.com/mdn/yari/pull/6464).

The best solution I could think of is to strip common parent paths from the displayed BCD keys (taking the last path item would not necessarily always work).

@queengooborg Would you be interested in having a look at this?

**Comment by queengooborg** [comment]
Rather than displaying the keys, what if we used the item descriptions instead?

**Comment by queengooborg** [comment]
Yeah that sounds like a good plan to me -- it seems like that's what is going on in the tables themselves, so consistency sounds good!

We should also show the description if there is one available?

**Comment by jpmedley** [comment]
>We should also show the description if there is one available?

Do you mean 'instead of the path' or 'in addition to the path'? I'd think the former, since the latter would be redundant.

**Comment by yarusome** [comment]
I think that it's good to also strip the last dot and whatever after it since they're already present as the row heads of BCD tables.
